microsoftgraph/microsoft-graph-explorer-v2
# Feat/theme

## Overview

Adds theming support for Graph Explorer

### Demo

#### Light
![light theme](https://user-images.githubusercontent.com/12011447/56336938-3485da00-61ab-11e9-825d-c38c7dd284b3.JPG)

#### Dark
![dark theme](https://user-images.githubusercontent.com/12011447/56336942-38196100-61ab-11e9-8320-9217e77b2dfe.JPG)

### Notes
This PR introduces a CSS-in-JS approach. From this point on, all styling should use this approach as the standard technique for styling. If you touch a scss file migrate it to CSS-in-JS.

Theme switching is manual right now, it can be automated once we find out how to detect the host document theme from the docs team.
## Testing Instructions

* Launch Graph Explorer
* Navigate to `src/index.tsx`
* In the `loadTheme` function change the argument to either dark or light
* Observe that GE's theme changes
